### PR TITLE
Add docker and update readme [resolves #13]

### LINKED ISSUE #13: chore: docker setup for local development
Docker setup with redis, database & adminer for local development.

### PR DESCRIPTION
**Pull request by Badhan-abhishek**

### CODE REVIEW

**Comment by Badhan-abhishek** [comment]
resolves #13

File: .env.example
Comment (viveknathani) [inline .env.example:1]: this bit feels redundant. can we revert this?

File: README.md [hunk 1]
```diff
@@ -21,6 +21,31 @@ Currently, I run this model's server locally on my computer with every PDF I can
...
+4. To run migrations
+    ```bash
+    yarn docker:migrate up # or "down"
+    ```
+5. To stop containers
+    ```bash
+    yarn docker:down
+    ```
+
+TODO: Create windows scripts
```
Comment (viveknathani) [inline README.md:47]: what do we need to do to complete this step?

**Review by viveknathani — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Badhan-abhishek) [reply README.md:47]: I don't have windows system to check if .bat scripts are working or not. That's why they are in TODO.

File: .env.example
Comment (Badhan-abhishek) [reply .env.example:1]: That's my formatter. I'll revert

Comment (Badhan-abhishek) [reply .env.example:1]: @viveknathani fixed

**Comment by viveknathani** [comment]
hi @Badhan-abhishek! appreciate all the effort put into this PR. in favour of [this issue](https://github.com/viveknathani/teachyourselfmath/issues/6) and [some recent development](https://github.com/viveknathani/teachyourselfmath/pull/21), i will be making some fundamental changes into how code is built and deployed for this project. the changes in this PR will therefore stop aligning. hence, i am closing this PR.

**Pull request closed without merging**